packages/ts/generator-typescript-core/src/Plugin.ts

```diff
@@ -1,5 +1,7 @@
-import type { Logger } from '@hilla/generator-typescript-utils/LoggerFactory.js';
+// eslint-disable-next-line import/no-duplicates
```

Comment:
Not big deal, but it hurts me to see a rule that would forbid a necessary language syntax.

The `no-duplicates` docs says:

> If the core ESLint version is good enough (i.e. you're not using Flow and you are using [import/extensions](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md)), keep it and don't use this.

Could that allow to avoid this thing that I consider a bug?

Reply:
🤔 Well, that's a good point. However, I also have some unnecessary dupes removed with this rule. Another option is to get rid of default imports completely in `utils` library (which also looks a nice option to me). What do you think?

Reply:
I don't know, `LoggerFactory.ts` looks fine as it is. TBH, my preferred option would be to keep things as they are and file a bug in `eslint-plugin-import`.